Tests/MapboxCoreNavigationTests/SimulatedLocationManagerTests.swift

```diff
@@ -1,19 +1,20 @@
 import XCTest
-import FBSnapshotTestCase
-import TestHelper
 import MapboxDirections
 @testable import MapboxCoreNavigation
-@testable import MapboxNavigation
+#if !SWIFT_PACKAGE
```

Comment:
If we don’t figure out #2866 first, make sure to at least run the tests locally using the Carthage-based target in Xcode. RouteControllerTests and SimulatedLocationManagerTests no longer run in an SPM-based test target, so the Carthage-based MapboxCoreNavigationTests is the only way to know whether these tests still pass.

Reply:
Thanks for the suggestion, I'll test these tests.